RM_SUI/RM_SUI/StoreTabBar/Tabs/ProductsView.swift

```diff
@@ -0,0 +1,145 @@
+// ProductsView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Products list
+struct ProductsView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let backgroundGradient: [Color] = [.yellow, .orange]
+        static let searchLabelName = "Search"
+        static let searchLabelImageName = "magnifyingglass"
+        static let filterButtonImageName = "slider.horizontal.3"
+        static let totalTextName = "Total:"
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        VStack {
+            navigationBarView
+
+            Text("\(Constants.totalTextName) \(productsViewModel.totalSum)")
+                .font(.title)
+                .fontWeight(.bold)
+                .foregroundColor(.red)
+
+            ScrollView(showsIndicators: false) {
+                ForEach(productsViewModel.products, id: \.name) { product in
+                    furnitureItemView(product: product)
+                }
+            }
+            .padding(.horizontal, 30)
+        }
+    }
+
+    // MARK: - Private Properties
+
+    @StateObject private var productsViewModel = ProductsViewModel()
```

Comment:
```suggestion
    @StateObject private var productViewModel = ProductViewModel()
```
Вроде в единственном числе нейминг